assets/oauth_clients/components/Clients.tsx

```diff
@@ -58,6 +58,7 @@ class Clients extends React.Component<any, any> {
 
         if (confirm(gettext('Would you like to delete client: {{name}}', {name: this.props.clientToEdit.name}))) {
             this.props.deleteClient('clients');
+            this.props.cancelEdit();
```

Comment:
The `async` branch should be reserved for backend changes only. We're trying to limit the frontend changes to none at all if possible.

This also looks like a common issue in the `develop` branch, so this fix might be better off going there (so it's in v2.8 release as well)

Reply:
would be good, to create a separate ticket for these issues, As I checked, these issues is also reproducing on nra-develop